release-notes/opensearch-dashboards-reports.release-notes-1.2.0.0.md

```diff
@@ -0,0 +1,40 @@
+### **Version 1.2.0.0 Release Notes**
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.2.0
+
+### **Bug Fixes**
+* Fix quoting and url-encoding ([#153](https://github.com/opensearch-project/dashboards-reports/pull/153))
+* Remove hard coded localhost when calling API ([#172](https://github.com/opensearch-project/dashboards-reports/pull/172))
+* rename plugin helper config file name to consistent with OSD ([#180](https://github.com/opensearch-project/dashboards-reports/pull/180))
```

Comment:
this is `maintenance`, it's to sync with upstream change

Reply:
I'll change it